native/src/contexts/__tests__/AppContextProvider.spec.tsx

```diff
@@ -96,6 +97,7 @@ describe('AppContextProvider', () => {
       jpalTrackingCode: '123456',
       jpalSignals: [],
       externalSourcePermissions: { 'https://vimeo.com': true },
+      selectedTheme: 'light' as ThemeKey,
```

Comment:
🔧 We want to avoid typecasts whenever possible. This becomes redundant if you change L88 to:
``` ts
    const settings: SettingsType = {
```